app/access/api.py

```diff
@@ -42,3 +45,30 @@ def users(request: HttpRequest) -> dict[str, list[UserSchema]]:
     models = User.objects.all()
     responses = [user_to_response(model) for model in models]
     return {"items": responses}
+
+
+@router.post("users", response={201: UserSchema})
+def create_user(request: HttpRequest, user_in: UserSchema) -> UserSchema:
+    """Create the given user and return it.
+
+    Return HTTP status code
+
+        - 201 (Created) if the User was created as expected
+        - 404 (Not Found) if there is no provider with the given provider ID
+        - 409 (Conflict) if there is already a record with the same username
+        - 422 (Unprocessable Content) if there is any other invalid value
+    """
+    provider = get_object_or_404(Provider, id=user_in.provider_id)
+
+    try:
+        user_out = User.objects.create(
+            username=user_in.username,
+            first_name=user_in.first_name,
+            last_name=user_in.last_name,
+            email=user_in.email,
+            provider=provider
```

Comment:
I guess for now this is good enough, but I suspect that in the future, the provider will be deduced from the user doing the operation. Then we'll have another UserSchema, one for the input. 🤔 But this is just my thinking way ahead

Reply:
:thinking: Good point, thanks. Created a ticket as a reminder for it [PB-1084](https://jira.swisstopo.ch/browse/PB-1084).

Reply:
@schtibe the provider of the user to create is not necessarily equal to the provider of the user that makes the call, e.g. if "we" or a platform admin (e.g. Mario) creates users

Reply:
@boecklic Aight, good to know!

Reply:
Also a good point :sweat_smile: Maybe having this is as an optional field would address both scenarios. Put it on the agenda to discuss this in person.